rune/libcontainer/specconv/spec_linux.go

```diff
@@ -347,6 +347,19 @@ func createEnclaveConfig(spec *specs.Spec, config *configs.Config) {
 		ra_type = sgx.InvalidRaType
 	}
 
+	raProductEnclave := filterOut(env, "ENCLAVE_RA_PRODUCT_ENCLAVE")
+	if raProductEnclave == "" {
+		raProductEnclave = libcontainerUtils.SearchLabels(config.Labels, "ra_product_enclave")
+	}
+	var ra_product_enclave uint32
+	if strings.EqualFold(raProductEnclave, "dev") {
```

Comment:
ENCLAVE_RA_PRODUCT_ENCLAVE is a boolean, so the possible value should be "true" or "false".